src/main/java/io/jenkins/plugins/opentelemetry/semconv/JenkinsOtelSemanticAttributes.java

```diff
@@ -75,6 +75,11 @@ public final class JenkinsOtelSemanticAttributes {
      * @see org.jenkinsci.plugins.workflow.graph.FlowNode#getId()
      */
     public static final AttributeKey<String> JENKINS_STEP_ID = AttributeKey.stringKey("jenkins.pipeline.step.id");
+    /**
+     * @see org.jenkinsci.plugins.workflow.pipelinegraphanalysis.GenericStatus
+     * @see org.jenkinsci.plugins.workflow.pipelinegraphanalysis.StatusAndTiming#computeChunkStatus2()
```

Comment:
This line is making javadoc to fail

```suggestion
```

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.6.0:javadoc (default-cli) on project opentelemetry: An error has occurred in Javadoc report generation: 
[ERROR] Exit code: 1
[ERROR] /Users/inifc/src/opentelemetry-plugin/src/main/java/io/jenkins/plugins/opentelemetry/semconv/JenkinsOtelSemanticAttributes.java:80: error: reference not found
[ERROR]      * @see org.jenkinsci.plugins.workflow.pipelinegraphanalysis.StatusAndTiming#computeChunkStatus2()
[ERROR]             ^
[ERROR] 1 error
[ERROR] Command line was: /usr/local/Cellar/openjdk/21.0.1/libexec/openjdk.jdk/Contents/Home/bin/javadoc @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/Users/inifc/src/opentelemetry-plugin/target/site/apidocs' dir.
```

Reply:
Happy New Year @kuisathaverat!

Thanks for the tip! Not sure how I missed that part of the logs... Fixed now -- I corrected the reference instead of removing it, as I think it does contain much useful info. The checks are passing now :)

Is there anything more needed for a merge?